src/server.js

```diff
@@ -139,29 +140,31 @@ class Server extends EventTarget {
       websockets = networkBridge.websocketsLookup(this.url);
     }
 
+    let normalizedData;
     if (typeof options !== 'object' || arguments.length > 3) {
       data = Array.prototype.slice.call(arguments, 1, arguments.length);
-      data = data.map(item => normalizeSendData(item));
+      normalizedData = data.map(item => normalizeSendData(item));
     } else {
-      data = normalizeSendData(data);
+      normalizedData = normalizeSendData(data);
     }
 
     websockets.forEach(socket => {
-      if (Array.isArray(data)) {
+      const messageData = socket instanceof SocketIO ? data : normalizedData;
```

Comment:
Before, a `_data` parameter was used. In this new setup `const _data` would issue a [no-underscore-dangle](https://eslint.org/docs/latest/rules/no-underscore-dangle) warning.